test/utils/fit-bounds.spec.js

```diff
@@ -2,45 +2,45 @@ import test from 'tape-catch';
 import {fitBounds} from 'react-map-gl';
 
 /* fitBounds from Mapbox for comparison */
-import Transform from 'mapbox-gl/js/geo/transform';
-import {LngLatBounds, Point} from 'mapbox-gl';
-import {toLowPrecision} from '../test-utils';
+// import Transform from 'mapbox-gl/js/geo/transform';
```

Comment:
How do we proceed here? Just create the expected result object manually and compare?